crates/hyperqueue/src/client/output/quiet.rs

```diff
@@ -0,0 +1,98 @@
+use std::path::Path;
+use std::time::Duration;
+use tako::common::resources::ResourceDescriptor;
+use tako::messages::common::WorkerConfiguration;
+use crate::client::job::WorkerMap;
+use crate::client::output::cli::task_status_to_cell;
+use crate::client::output::outputs::Output;
+use crate::client::status::{job_status, Status};
+use crate::common::serverdir::AccessRecord;
+use crate::server::autoalloc::{Allocation, AllocationEventHolder};
+use crate::server::job::{JobTaskCounters, JobTaskInfo};
+use crate::stream::reader::logfile::Summary;
+use crate::transfer::messages::{AutoAllocListResponse, JobDetail, JobInfo, LostWorkerReasonInfo, StatsResponse, WaitForJobsResponse, WorkerExitInfo, WorkerInfo};
+use crate::WorkerId;
+
+
+fn to_str(stat: &Status) -> anyhow::Result<&str> {
+    Ok(match stat {
+        Status::Waiting => "WAITING",
+        Status::Running => "RUNNING",
+        Status::Finished => "FINISHED",
+        Status::Failed => "FAILED",
+        Status::Canceled => "CANCELED",
+        _ => anyhow::bail!("Invalid job status"),
+    })
+}
+
+
+#[derive(Default)]
+pub struct Quiet;
+
+impl Output for Quiet {
+    // Workers
+    fn print_worker_list(&self, workers: Vec<WorkerInfo>) {
+        for worker in workers {
+            let worker_status = match worker.ended.as_ref() {
+                None => "RUNNING",
+                Some(WorkerExitInfo {
+                         reason: LostWorkerReasonInfo::ConnectionLost,
+                         ..
+                     }) => "CONNECTION LOST",
+                Some(WorkerExitInfo {
+                         reason: LostWorkerReasonInfo::HeartbeatLost,
+                         ..
+                     }) => "HEARTBEAT LOST",
+                Some(WorkerExitInfo {
+                         reason: LostWorkerReasonInfo::IdleTimeout,
+                         ..
+                     }) => "IDLE TIMEOUT",
+                Some(WorkerExitInfo {
+                         reason: LostWorkerReasonInfo::Stopped,
+                         ..
+                     }) => "STOPPED",
+            };
+            println!("worker {} is {}", worker.id, worker_status)
```

Comment:
The output should be machine parsable as easily as possible, so let's not output things like `"worker .. is "` etc.
Just output a line for each worker with the worker ID and status, separated by a space.